pkg/subctl/cmd/utils/utils.go

```diff
@@ -94,3 +99,32 @@ func GetSubmarinerResource(config *rest.Config) *v1alpha1.Submariner {
 
 	return submariner
 }
+
+// IsValidCIDR validates the supplied string is a valid v4/v6 network address
+func IsValidCIDR(cidr string) error {
```

Comment:
This function is already defined in _pkg/discovery/globalnet/globalnet.go_.

Reply:
Thanks Tom, I was thinking moving it to a utils package made sense but didn't realize it was going away. If cool, I will import from `pkg/discovery/globalnet/`.